scripts/allowlist/genesisContent/besu_genesis.json

```diff
@@ -0,0 +1,58 @@
+{
+	"config": {
+		"chainid": 2017,
+		"homesteadBlock": 0,
+		"eip150Block": 0,
+		"eip155Block": 0,
+		"eip158Block": 0,
+		"byzantiumBlock": 0,
+		"qbft": {
+			"block": 100,
+			"epochlength": 30000,
+			"blockperiodseconds": 5,
+			"requesttimeoutseconds": 10,
+			"validatorcontractaddress": "0x0000000000000000000000000000000000008888"
```

Comment:
nit: should we use a different default value than the permissioning smart contracts? or is this deliberate?

Reply:
9999 is also used for permissioning!

Reply:
you can have 7777 ?